source/Cosmos.System2/Keyboard/ScanMaps/US_Standard.cs

```diff
@@ -60,8 +60,8 @@ protected override void InitKeys()
             _keys.Add(new KeyMapping(0x1A, '[', '{', '[', '{', '[', '{', ConsoleKeyEx.LBracket));
             _keys.Add(new KeyMapping(0x1B, ']', '}', ']', '}', ']', '}', ConsoleKeyEx.RBracket));
             /* ENTER, CTRL */
-            _keys.Add(new KeyMapping(0x1C, ConsoleKeyEx.Enter));
-            _keys.Add(new KeyMapping(0x1D, ConsoleKeyEx.LCtrl));
+            _keys.Add(new KeyMapping(0x9C, ConsoleKeyEx.Enter));
```

Comment:
I think this is causing the test to fail since this PR breaks pressing enter in the console no longer works. See https://github.com/CosmosOS/Cosmos/blob/9d59ffa7bc06dcbe3268cd5906ce9e69c500a438/Tests/Kernels/Cosmos.Compiler.Tests.SingleEchoTest/Kernel.cs#L30 for the reason the test is failing.